P/Packmol/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Packmol"
+version = v"20.3.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://leandro.iqm.unicamp.br/m3g/packmol/packmol.tar.gz", "21c249670e08dfed142658030ea58bc4473483a4b9da263abff13588f3c47f31")
```

Comment:
This looks like an unversioned tarball?  So a new release will break this recipe?

Reply:
This seems to be a much better option: https://github.com/m3g/packmol/archive/refs/tags/v20.3.5.tar.gz

Reply:
I did not know there is a github repo yesterday.